app/Lib/lang.php

```diff
@@ -1287,6 +1287,7 @@
   'er.ug.version' =>  'Unknown version "%1$s"',
   'er.unknown' =>     'Unknown value "%1$s"',
   'er.unknown.def' =>  'Unknown value',
+  'er.unknown.er' =>  'An unknown error occurred.',
```

Comment:
How do we have an unknown error? Don't we control everything here? (If I'm an admin, what do I do with this? If I'm a COmanage developer, how do I debug it?)

Reply:
This was actually inserted to represent an HTTP 500 error that could occur during the ajax call and bubbled back up to the browser. I've changed this to be more explicit:
`'er.500' =>         'Internal server error (500)'`